rpc/handlers.go

```diff
@@ -327,6 +329,40 @@ func (h *Handler) BlockWithTxs(id BlockID) (*BlockWithTxs, *jsonrpc.Error) {
 	}, nil
 }
 
+func (h *Handler) BlockWithReceipts(id BlockID) (*BlockWithReceipts, *jsonrpc.Error) {
```

Comment:
I think we need to return l1_gas and l1_data_gas in Execution resources ([spec](https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L4019) vs [juno](https://github.com/NethermindEth/juno/blob/main/rpc/transaction.go#L302)).

Reply:
I think it's done in recent commit